bindings/nodejs/lib/Client.ts

```diff
@@ -0,0 +1,287 @@
+// Copyright 2021-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+import { MessageHandler } from './MessageHandler';
+import type {
+    Address,
+    ClientOptions,
+    GenerateAddressesOptions,
+    NodeInfo,
+    OutputResponse,
+} from '../types';
+import type { Message } from '../types/message';
+import type { MessageMetadata } from '../types/messageMetadata';
+import type { GenerateMessageOptions } from '../types/generateMessageOptions';
+import type { QueryParameter } from '../types/queryParameters';
+import type { UTXOInput } from '../types/inputs/UTXOInput';
+import type { Payload } from '../types/payloads';
+import type { PreparedTransactionData } from '../types/preparedTransactionData';
+
+export class Client {
+    private messageHandler: MessageHandler;
+
+    constructor(options: ClientOptions) {
+        this.messageHandler = new MessageHandler(options);
+    }
+
+    async getInfo(): Promise<NodeInfo> {
+        const response = await this.messageHandler.callClientMethod({
+            name: 'GetInfo',
+        });
+
+        return JSON.parse(response).payload;
+    }
+
+    /**
+     * Gets the network related information such as network_id and min_pow_score
+     */
+    async getNetworkInfo() {
+        const response = await this.messageHandler.callClientMethod({
+            name: 'GetNetworkInfo',
+        });
+
+        return JSON.parse(response).payload;
+    }
+
+    // TODO: proper type for queryParameters
+    async outputIds(queryParameters: QueryParameter[]): Promise<string[]> {
```

Comment:
Are these defined in [iota.js ](https://github.com/iotaledger/iota.js/tree/feat/stardust)? If yes, we should just use them.

Reply:
I forgot to remove this TODO, I did define this type. As far as I can see in Iota.js it is typed as string[], and in the jsdoc params here: https://github.com/iotaledger/iota.js/blob/feat/stardust/packages/iota/src/clients/plugins/indexerPluginClient.ts#L36-L65. 

I based my type (https://github.com/oddgrd/iota.rs/blob/feat/node-ts/bindings/nodejs/types/queryParameters/index.ts) on the rust type.